papers.md

```diff
@@ -0,0 +1,207 @@
+
+#### 2. Open an issue and announce the writing of the paper
+
+The Coordinator should open an issue to organize the writing of the paper on
+that repository and then announce to the Jupyter Google Group that the paper is
+being worked on and that anyone can contribute (with a link to the issue).
+
+#### 3. Draft the paper in the repository
+
+At this point, the Coordinator should organize the writing of the actual paper.
+
+The [JOSS Author Guidelines](http://joss.theoj.org/about) detail the format and
+requirements of a JOSS paper. In general this will involve creating a `paper`
+subdirectory in the root of the repository with two files: `paper.md` and
+`paper.bib`. The paper should be written using our standard processes for pull
+requests and code review.
+
+#### 3. Email potential authors
+
+Once a final draft of the paper has been merged into the repository, the
+Coordinator should email all potential authors of the paper, inviting them to
+participate. Usually this will include at least two emails:
+
+* Individually email all contributors listed in the Git log of the repository.
+  This list can be generated using the Git command `git log --all --format='%cN
+  <%cE>' | sort -u`.
+* Email the main Jupyter Google Group to include folks not listed in the Git
+  log.
+
+Both of these emails should include:
+
+* A verbatim copy of the above JOSS authorship criteria;
+* A description the tasks each individual needs complete to be an author on the
+  paper;
+* A concrete deadline for completing those tasks (at least 2 weeks out); and
+* A link to the paper's markdown file in the repository.
+
+The tasks each individual needs to complete are as follows:
+
+* Create an account on [ORCID](http://orcid.org/).
+* Submit a PR to the repository adding their name, ORCID id, and affiliation to
+  the paper;
+* Provide any edits to the content of the paper;
+* In the comment of the PR, briefly describe their contributions to the work;
+  and
+* In the comment of the PR, provide verification that they have read the paper
+  and agree to its publication.
+
+As the deadline approaches, the Coordinator should remind potential authors to
+complete their tasks.
+
+#### 4. Final submission
+
+Once the deadline for authors to complete their tasks has passed, the
+Coordinator should make sure authors are listed in alphabetical order. The
+Coordinator can then submit the paper.
+
+### Process for traditional academic papers
+
+Papers submitted to other (non-JOSS) journals, will usually be longer and take
+a more significant amount of time to author. They will also usually be authored
+in dedicated repository (under the jupyter-resources GitHub organization).
+Because of these factors, the process is slightly different.
+
+### Authorship criteria
+
+The authorship criteria for non-JOSS papers is slightly different than for JOSS
+in that all authors are expected to actively participate in the writing and
+editing of the paper:
```

Comment:
I felt this way before seeing how the SymPy paper went. They had almost 30 authors and it was by far the best group paper writing experience I have ever seen. Granted, not all 30 shared the writing and editing equally (a few people did most of the work, with others doing editing and discussing the content). But not requiring a co-author to do any active work on a paper is the wrong message and I think in SymPy's case, not requiring authors to actively participate would have led to a lesser outcome.

You are right that a 500 person paper would need to be handled differently, but I don't think we are going to face that. For use I think the few dozen range is more likely and would be covered by this. 

To cover these things, I will add some language to clarify that "actively participate" can be interpreted in a flexible way depending on the number of authors.